mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/JPAMailboxManagerTest.java

```diff
@@ -1,56 +1,56 @@
-    
-    @Override
-    protected MailboxManager provideMailboxManager() {
-        if (!openJPAMailboxManager.isPresent()) {
-            openJPAMailboxManager = Optional.of(JpaMailboxManagerProvider.provideMailboxManager(JPA_TEST_CLUSTER));
-        }
-        return openJPAMailboxManager.get();
-    }
-
-    @AfterEach
-    void tearDownJpa() {
-        JPA_TEST_CLUSTER.clear(JPAMailboxFixture.MAILBOX_TABLE_NAMES);
-    }
-
-    @Ignore("MAILBOX-343")
-    @Test
-    @Override
-    public void creatingConcurrentlyMailboxesWithSameParentShouldNotFail() {
-
-    }
-
-}
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox.jpa;
+
+import java.util.Optional;
+
+import org.apache.james.backends.jpa.JpaTestCluster;
+import org.apache.james.mailbox.MailboxManager;
+import org.apache.james.mailbox.MailboxManagerTest;
+import org.apache.james.mailbox.jpa.openjpa.OpenJPAMailboxManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class JPAMailboxManagerTest extends MailboxManagerTest {
+
+    private static final JpaTestCluster JPA_TEST_CLUSTER = JpaTestCluster.create(JPAMailboxFixture.MAILBOX_PERSISTANCE_CLASSES);
+    private Optional<OpenJPAMailboxManager> openJPAMailboxManager = Optional.empty();
+    
+    @Override
+    protected MailboxManager provideMailboxManager() {
+        if (!openJPAMailboxManager.isPresent()) {
+            openJPAMailboxManager = Optional.of(JpaMailboxManagerProvider.provideMailboxManager(JPA_TEST_CLUSTER));
+        }
+        return openJPAMailboxManager.get();
+    }
+
+    @AfterEach
+    void tearDownJpa() {
+        JPA_TEST_CLUSTER.clear(JPAMailboxFixture.MAILBOX_TABLE_NAMES);
+    }
+
+    @Disabled("MAILBOX-343")
```

Comment:
Seem https://issues.apache.org/jira/browse/MAILBOX-343 is resolved, right? If `creatingConcurrentlyMailboxesWithSameParent` is not supported in `JPAMailboxManager`, could you add a description to explain it to avoid miss understanding?